modules/kotest/src/test/kotlin/org/testcontainers/containers/TestContainerKotestIntegrationPerTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.testcontainers.containers
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+import kotlinx.coroutines.delay
+
+class TestContainerKotestIntegrationPerTest : StringSpec({
+    val container = GenericContainer<Nothing>("ubuntu")
+    setContainerCommand(container) // Set container command for first time to echo hello Kotest
```

Comment:
Since the integration point is `Startable` interface, you can just implement the interface and record `start()`/`stop()` calls

Reply:
I kept that GenericContainer because our initial requirement was for DockerContainer.

Reply:
yes, but it is not needed anymore and the test will be much simpler.

Reply:
ok

Reply:
done